ada_description/urdf/ada.xacro

```diff
@@ -23,7 +23,7 @@
   <link name="root"/>
   
   <link name="root_tilt"/>
-  <joint name="robot_tilt" type="revolute">
+  <joint name="robot_tilt" type="fixed">
```

Comment:
Does this break the code when the IMU is plugged in? I'm not sure how the TF will respond when a joint it was told is fixed has transforms published to it.

Reply:
Setting the base link to `j2n6s200_link_base` instead of `root` (which means the KDL chain excludes this joint anyway) fixes the issues we say before. So I turned this back to revolute.